src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -126,7 +126,7 @@ def load_arguments(self, _):
         c.argument('name', options_list=['--name', '-n'], help='name of the new web app',
                    validator=validate_site_create,
                    local_context_attribute=LocalContextAttribute(name='web_name', actions=[LocalContextAction.SET],
-                                                                 scopes=['webapp']))
+                                                                 scopes=['webapp', 'cupertino']))
```

Comment:
@arrownj Can you also add 'cupertino' to the command "webapp up"? It turns out in some tutorials they are using "webapp up" to create webapp.